lib/formatters/table.js

```diff
@@ -9,13 +9,22 @@
 //------------------------------------------------------------------------------
 
 const chalk = require("chalk"),
-    table = require("table").table,
-    pluralize = require("pluralize");
+    table = require("table").table;
 
 //------------------------------------------------------------------------------
 // Helpers
 //------------------------------------------------------------------------------
 
+/**
+ * Given a word and a count, append an "s" if count is not one.
+ * @param {string} word A word.
+ * @param {number} count A number.
```

Comment:
The description of parameter `count` can be improved.
```suggestion
 * @param {number} count Quantity.
```